src/verify_args.cpp

```diff
@@ -42,9 +44,9 @@ bool verify_args(const std::string& name,
             // TODO: Check for nans
             std::cout << "FAILED: " << name << std::endl;
             std::cout << "RMS Error: " << rms_error << std::endl;
-            if(ref.size() < 32)
+            if(ref.size() < 32 or enabled(MIGRAPHX_VERIFY_DUMP_DIFF{}))
```

Comment:
Suggestion with a little bit nicer formatting and making sure both outputs are displayed together:
```
            if((ref.size() < 32 and target.size() < 32) or enabled(MIGRAPHX_VERIFY_DUMP_DIFF{}))
            {
                std::cout << "ref: " << ref << std::endl << std::endl;
                std::cout << "target: " << target << std::endl;
            }
            else   
                std::cout << "(Use env MIGRAPHX_VERIFY_DUMP_DIFF to display non-matching results)" << std::endl;
```